bl.js

```diff
@@ -1,4 +1,4 @@
-var DuplexStream = require('readable-stream/duplex')
+var DuplexStream = require('stream').Duplex || require('readable-stream').Duplex
```

Comment:
Unfortunately no. It should be `require('readable-stream').Duplex` if you want to change this.
This is not really a fallback, but a dependency in the true sense.